artifactory_cleanup/cli.py

```diff
@@ -53,6 +53,14 @@ class ArtifactoryCleanupCLI(cli.Application):
         envname="ARTIFACTORY_CLEANUP_DESTROY",
     )
 
+    _ignore_not_found = cli.Flag(
+        "--ignore-not-found",
+        help="Ignores 404 errors when deleting artifacts",
+        mandatory=False,
+        default=False,
+        envname="IGNORE_NOT_FOUND",
```

Comment:
Let's start all env variables with the prefix
```suggestion
        envname="ARTIFACTORY_CLEANUP_IGNORE_NOT_FOUND",
```